src/achievements.cpp

```diff
@@ -91,6 +64,39 @@ void achievements::process_achievements_file(const config& cfg, const std::strin
 	}
 }
 
+/**
+ * Reads the mainline achievements.cfg and then all the achievements of each installed add-on.
+ *
+ * This is intentionally handled separately from other WML loading so that:
+ * a) All achievements and their status are able to be displayed on the main menu right after Wesnoth starts and regardless of which add-ons are active.
+ * b) Add-ons can add additional achievements to other content, whether UMC or mainline. For example, a modification that adds more achievements for mainline campaigns.
+ *
+ * NOTE: These are *not* in any way related to Steam achievements!
```

Comment:
This line should probably still be in the class / constructor doc instead of moving to reload().

Reply:
Updated.